text/0093-metric-instruments.md

```diff
@@ -0,0 +1,157 @@
+as well, to support built-in value types (e.g., timestamps).
+
+### Delta, Cumulative, and Instantaneous instruments
+
+Instruments can be categorized as to whether their values are "Delta",
+"Cumulative", or "Instantaneous".  This depends on which refinements
+are present and is required knowledge for exporters of the Sum-only
+instruments, since they may need to convert back and forth between
+delta-values and cumulative-values.  The categorization is determined
+as follows:
+
+Instruments with the Sum-only refinement, but without the
+Precomputed-sum refinement, produce delta instruments.
+
+Instruments with the Sum-only and Precomputed-sum refinements produce
+cumulative instruments.
+
+Instruments without the Sum-only refinement produce instantaneous
+measurements.
+
+## Internal details
+
+The existing specification includes three instruments: Counter,
+Measure, and Observer.  In this proposal, the two foundational
+instruments, Measure and Observer (synchronous and asynchronous,
+respectively), become abstract names, in the sense that all
+synchronous instruments are instances of a Measure instrument and all
+asynchronous instruments are instances of an Observer instrument.  New
+instrument names are given to the unrestricted, unrefined versions of
+the foundational instruments.  The following table summarizes this
+proposal:
+
+| Existing name | Proposed name | Sync or Async | Refinements | Measurement kind |
+| ------------- | ------------- | ------------- | ----------- | ---------------- | 
+| Counter       | Counter       | Sync          | Sum-only, Non-negative, Non-negative-rate | Delta |
+|               | UpDownCounter | Sync          | Sum-only | Delta |
+| Measure       | Distribution  | Sync          | _None_  | Instantaneous |
+|               | Timing        | Sync          | Non-negative, correct Duration units  | Instantaneous |
+| Observer      | LastValueObserver | Async     | _None_ | Instantaneous |
+|               | DeltaObserver | Async         | Sum-only, Non-negative, Non-negative-rate | Delta |
+|               | CumulativeObserver | Async    | Sum-only, Precomputed-sum, Non-negative-rate | Cumulative |
+
+### Synchronous instruments
+
+Two new synchronous instruments are introduced in this proposal, bringing the total to four.
+
+1. **Counter** remains unchanged.  It uses Sum aggregation by default.
+2. **Distribution** is an an unrefined Measure instrument.  Distribution accepts positive or negative values and uses MinMaxSumCount aggregation by default.
+3. **UpDownCounter** is a Sum-only instrument with no other refinements.  It supports capturing positive and negative changes to a sum (deltas).  UpDownCounter uses Sum aggregation by default.
+4. **Timing** is a Non-negative instrument specialized for the native clock duration measured on the platform.  It ensures that duration values are always captured with correct units, that ensures exporters can convert duration measurements correctly.
```

Comment:
I 100% think we should include this as an instrument, but having a unit based refinement wasn't specified at the start of the OTEP. Can we add something there about this?